scripts/setup-adapters.sh

```diff
@@ -54,33 +54,26 @@ function install_aws_deps {
 }
 
 function install_gcs-sdk-cpp {
-  # Install gcs dependencies
+  # Install gcs dependencies that cannot be installed via cmake's FetchContent
   # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md#required-libraries
 
-  # abseil-cpp
-  github_checkout abseil/abseil-cpp 20240116.1 --depth 1
-  sed -i 's/^#define ABSL_OPTION_USE_\(.*\) 2/#define ABSL_OPTION_USE_\1 0/' "absl/base/options.h"
-  cmake_install -DBUILD_SHARED_LIBS=OFF \
-    -DABSL_BUILD_TESTING=OFF
-
   # crc32
   github_checkout google/crc32c 1.1.2 --depth 1
-  cmake_install -DBUILD_SHARED_LIBS=OFF \
+  cmake_install \
+    -DBUILD_SHARED_LIBS=OFF \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=OFF \
     -DCRC32C_BUILD_TESTS=OFF \
     -DCRC32C_BUILD_BENCHMARKS=OFF \
     -DCRC32C_USE_GLOG=OFF
 
   # nlohmann json
   github_checkout nlohmann/json v3.11.3 --depth 1
-  cmake_install -DBUILD_SHARED_LIBS=OFF \
+  cmake_install \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=OFF \
+    -DBUILD_TESTING=OFF \
     -DJSON_BuildTests=OFF
-
-  # google-cloud-cpp
-  github_checkout googleapis/google-cloud-cpp v2.22.0 --depth 1
-  cmake_install -DBUILD_SHARED_LIBS=OFF \
-    -DCMAKE_INSTALL_MESSAGE=NEVER \
-    -DGOOGLE_CLOUD_CPP_ENABLE_EXAMPLES=OFF \
-    -DGOOGLE_CLOUD_CPP_ENABLE=storage
```

Comment:
Why remove this? We want to install all deps into the docker image (or locally), even if they are in resolve_dependency.

Reply:
I was not aware that it was being used to create the image, I will add it back.